recipes/management/increasing_the_capacity_to_win/growing/expectations/README.md

```diff
@@ -1,2 +1,33 @@
+# Expectations
+Expectations are what most managers keep hidden in deep, impregnable underground siloes that grind open painfully slow once a year, during review season, to nuke any last vestige of hope that you might get a raise next year. It's a popular approach that gives them lots of optionality in exchange for very little effort. 
+
+Here, we provide a variety of recipes on how you can set expectations with your engineers. These recipes often become ingredients to other parts of 'Increasing the Capacity To Win', for example the compensation recipes, coaching recipes, mentorship recipes, etc. 
+
+# Table of Contents
+* [Scorecards](scorecards/README.md): the most detailed type of expectations, usually tailor-made for a given role.
+* [Ladders](ladders/README.md): expectations for a given 'job family' and how they change at increasing 'levels'. 
+* [Onboardings](onboarding/README.md): specialized expectations targeting employees new to a role. 
+
+
+# Explaining 'Expectations' Recipes
+
+## Dobromir's Recipe
+These are raw notes right now. Will turn into a recipe in the future.
+
+* Defining 'winning' is almost half the battle!
+* Most great people will figure out how to win, if they understand the definition. Aligning on it is the hard part?
+* Focus on RESULTS (outputs) not on APPROACH (input). 
```

Comment:
I'm not sure I agree with outputs / inputs - it's outputs vs method, or what vs how